app/src/@types/Settings.d.ts

```diff
@@ -6,6 +6,7 @@ interface Settings {
     vimcEmail: string;
     slackUrl: string;
     isApplicantTouchstone: (id: string) => boolean;
+    is201804ApplicantTouchstone: (id: string) => boolean;
```

Comment:
I know this is just temporary, but maybe it could be `isStochasticApplicantTouchstone` and then later we could make the logic more general?